test/e2e/features/pageObjects/caseCreatedPage.js

```diff
@@ -12,9 +10,7 @@ class CaseCreatedPage extends BasePage{
   }
 
   async amOnPage(){
```

Comment:
If this method is synchronous please remove async. If its async please add await keyword